index.html

```diff
@@ -115,10 +200,32 @@ <h2 class="post">Multi-Post <br>
                 <button type="button" class="button">See Project</button>
             </div>
             </div>
-         
+            <div class="block5">
+                <h2 class="post">Uber Navigation </h2>
+                    <div class="frame">
+                        <img src="images/frame3.png" alt="tabs">
+                    </div>
+            <p class="content-text-px">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+            <div class="tags">
+                <div class="tags1">
+                <ul>
+                    <li><a href="#" class="tag-item1">html</a></li>
+                    <li><a href="#" class="tag-item1">Ruby on Rails</a></li>
+                    <li><a href="#" class="tag-item1">css</a></li>
+                    <li><a href="#" class="tag-item1">javascript</a></li>
+                </ul>
+            </div></div>
+            <div class="action">
+                <button type="button" class="button">See Project</button>
+            </div>
+            </div>
+         </div>
+         </div>
         </div>
         <div>
             <div class="content-x">
+                <div class="content-x-y">
+                    <div id="about1">
```

Comment:
Kindly make your about section also responsive and allow it to take the full width of the screen. Kindly also fix the padding and spacing around and between elements to make it look like the Figma design 🤗
Also, note that the left part of the About section should take 40% of the screen's width and the right part the remaining 60% of the screen's width.
Would also be nice to add the curved top corner as I mentioned below 😃

Your design:

![image](https://user-images.githubusercontent.com/73714615/188265952-5a151378-bd25-4cf4-8d73-108fbf374e44.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188265991-86499bee-47d1-4e72-a20b-0e6564ec076c.png)

Reply:
- Kindly, I recommend you follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉
- Kindly, I suggest you implement the rounded corner, also adapt this section to 100% of the width of your page.